validator/rpc/wallet.go

```diff
@@ -252,6 +262,20 @@ func (s *Server) ChangePassword(ctx context.Context, req *pb.ChangePasswordReque
 	if !valid {
 		return nil, status.Errorf(codes.FailedPrecondition, invalidWalletMsg)
 	}
+	if req.CurrentPassword == "" {
+		return nil, status.Error(codes.InvalidArgument, "Current wallet password cannot be empty")
+	}
```

Comment:
Nitpick alert

I am not sure it makes much sense to do any additional checks except comparing the current password with the one from disk. We disallow empty passwords in the first place anyway.